contrib/win32/win32compat/pwd.c

```diff
@@ -236,9 +236,11 @@ get_passwd(const wchar_t * user_utf16, PSID sid)
 		goto cleanup;
 	}
 
-	/* If standard local user name, just use name without decoration */
-	if ((_wcsicmp(domain_name, computer_name) == 0) && (_wcsicmp(computer_name, user_name) != 0))
-		wcscpy_s(user_resolved, ARRAYSIZE(user_resolved), user_name);
+	/* if standard local user name or system account, just use name without decoration */
+	const SID_IDENTIFIER_AUTHORITY nt_authority = SECURITY_NT_AUTHORITY;
```

Comment:
1)If statement is missing body.
wcscpy_s(user_resolved, ARRAYSIZE(user_resolved), user_name);

2) please use () as there are multiple &&, || conditions here. It improves readability.

Reply:
Sorry about missing the body, it was a copy/paste error between branches.

Great point about using additional () to increase the readability.